### ISSUE TITLE
Add translation support

### ISSUE THREAD

**Issue by rene-coty** [CleoMenezesJr/auto-activities#23]
It would be great to be able to translate this extension into several languages (at least the Preferences window)..

**Comment by CleoMenezesJr** [comment]
Thanks, coincidentally I was thinking about this just today.

**Comment by Vistaus** [comment]
I agree, and I will provide the Dutch translation once this is implemented. 🙂
